### PR TITLE
Remove charset restrictions on tag values

### PR DESCRIPTION
**Pull request by ramonza**
See: https://github.com/census-instrumentation/opencensus-specs/pull/71

### CODE REVIEW

File: tag/validate.go [hunk 1]
```diff
@@ -52,5 +52,5 @@ func checkValue(v string) bool {
 	if len(v) > maxKeyLength {
 		return false
 	}
-	return isASCII(v)
+	return true
```
Comment (odeke-em) [inline tag/validate.go]: In this case we can fold this entirely into
```go
func checkValue(v string) bool {
        return len(v) <= maxKeyLength
}
```

File: tag/map_test.go [hunk 2]
```diff
@@ -129,9 +129,7 @@ func TestNewMap(t *testing.T) {
 			name:    "from empty; invalid",
 			initial: nil,
 			mods: []Mutator{
-				Insert(k5, "v\x19"),
-				Upsert(k5, "v\x19"),
-				Update(k5, "v\x19"),
+				Insert(k5, strings.Repeat("x", 256)),
```
Comment (odeke-em) [inline tag/map_test.go]: Perhaps this could be a test of its own since the restriction being newly tested is `len(v) <= 255`

Comment (ramonza) [reply tag/map_test.go]: added t.Run, so it should appear as it's own test

Comment (odeke-em) [reply tag/map_test.go]: Cool, thanks. LGTM!

**Pull request merged**